### PR TITLE
right side bar frontend basic

### PR DESCRIPTION
**Pull request by henrywangzh**

### CODE REVIEW

**Review by olpai — APPROVED** [review]
lgtm

**Review by justinsheu — APPROVED** [review]
lgtm

File: src/App.tsx
Comment (jason2020) [inline src/App.tsx:1]: This is where the Github Actions for the CD build is failing, please look into the following comments by GitHubActions.

File: src/components/ClassGraph.tsx
Comment (jason2020) [inline src/components/ClassGraph.tsx:1]: There are many TS-related typing errors that are breaking the build

**Review by jason2020 — CHANGES_REQUESTED** [review]
Changes seem fine, but need to fix CI:CD

**Review by jason2020 — APPROVED** [review]
LGTM

**Pull request merged**